source/MaterialXGraphEditor/Graph.cpp

```
@@ -2753,6 +2753,10 @@ void Graph::deleteLinkInfo(int startAttr, int endAttr)
     int upNode = getNodeId(startAttr);
     int downNode = getNodeId(endAttr);
 
+    if (upNode == -1 || downNode == -1) {
```

Should we keep this? Or just remove it and let it crash when bad things happen